.travis.yml

```diff
@@ -14,12 +14,22 @@ cache:
 before_script:
   - (cd admin && ./init.sh)
 
+stages:
+  - name: build
```

Comment:
maybe add a comment that the `build` stage is not excluded on `pull_request` because it has a (parallel) job for the spec

Reply:
ok